docs/index.html

```diff
@@ -7,13 +7,94 @@
     <title>TinDog</title>
     <link rel="stylesheet" href="./styles/styles.css">
     <script src="https://code.jquery.com/jquery-3.6.0.js" integrity="sha256-H+K7U5CnXl1h5ywQfKtSj8PCmoN9aaq30gDh27Xc0jk=" crossorigin="anonymous"></script>
-    <style>#sidenav {display: none;}</style>
+    <style>#sidenav {display: none;}
+           #slideshow {
+                float: right;
+                position: relative;
+                margin-top: -4em;
+                margin-right: 5em;
+                margin-left: 10em;
+                width: 24em;
+            }
+            @media only screen and (max-width: 896px) {
+                #slideshow {
+                    margin: auto auto;
+                    float: none;
+                    position: relative;
+                    width: 14em;
+                }
+            }
+            #slideshow img {
+                    width: inherit;
+                    border: solid 5px #fd9eb9;
+                    border-radius: 1em;
+                    background-color: #fe3c72;
+            }
+
+            .overlay {
+            position: absolute;
+            bottom: 0;
+            background: rgba(0, 0, 0, 0.5); /* Black see-through */
+            color: #f1f1f1;
+            width: 90%;
+            margin-left: 5%;
+            transition: .5s ease;
+            opacity:0;
+            font-size: 20px;
+            padding: 20px 5px;
+            margin-bottom: 1em;
+            text-align: center;
+            border-radius: 1em;
+            }
+
+            #slideshow:hover .overlay, #slideshow:focus .overlay {
+                opacity: 1;
+            }
+            #imgdiv {
+                padding-left: 0;
+                position: relative; 
+                height: 100px;
+                width: 100%;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+            }
+            #imgdiv::before {
+                background: linear-gradient(rgba(0, 0, 0, 0.5), 
+                                            rgba(0, 0, 0, 0.5) ), 
+                                            url('./imgs/banner.jpeg');
+                background-size: cover;
+                position: absolute;
+                content: "";
+                top: 0px;
+                right: 0px;
+                bottom: 0px;
+                left: 0px;
+                opacity: 0.60;
+            }
+            h1 {
+                position: relative;
+                color: #fff;
+                font-size: xx-large;
+                text-align: center;
+                
+            }
```

Comment:
This block (and the @media tag following it) are the title bar - for coherent formatting this should be in every file as well